packages/calendar/src/components/input-types/date-text-input/DateTextInput.stories.tsx

```diff
@@ -110,9 +101,15 @@ storiesOf("calendar/Input/DateTextInput", module)
       <DateTextInput
         value={store.state.value}
         onValueChange={value => store.set({ value })}
-        textInputTheme={createDateTextInputTheme({
-          backgroundColor: "lightgoldenrodyellow",
-          backgroundColorInvalidDate: "lightcoral"
+        theme={createDateTextInputTheme({
+          textInput: createStandardTextInputTheme({
+            backgroundColor: color("textInput-backgroundColor", "#fafad2"),
+            backgroundColorInvalid: color(
+              "textInput-backgroundColorInvalid",
```

Comment:
We have used natural language for all other knobs.
Maybe just "Background color when invalid"?